### PR TITLE
Library items for Coin API tutorial

### PR DESCRIPTION
**Pull request by srosam**

### CODE REVIEW

File: python/transformations/Sentiment-Analysis/library.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "DeploySettings":
     {
         "DeploymentType": "Service",
-        "CpuMillicores": 200,
+        "CpuMillicores": 1000,
```
Comment (tbedford) [inline python/transformations/Sentiment-Analysis/library.json:37]: This is for sentiment analysis, OK as long as this is intentional.

File: python/transformations/Threshold-Alert/README.md [hunk 2]
```diff
@@ -16,6 +17,7 @@ The code sample uses the following environment variables:
 - **output**: This is the output topic for alerts.
 - **ParameterName**: This is the stream's parameter to track.
```
Comment (tbedford) [inline python/transformations/Threshold-Alert/README.md:18]: Uses camelcase in the code.

File: python/transformations/Threshold-Alert/README.md [hunk 3]
```diff
@@ -16,6 +17,7 @@ The code sample uses the following environment variables:
...
 - **ThresholdValue**: This is the threshold's numerical value.
```
Comment (tbedford) [inline python/transformations/Threshold-Alert/README.md:19]: Uses camelcase in the code.

**Review by tbedford — APPROVED** [review]
A few minor comments, but LGTM!

**Review by tbedford — APPROVED** [review]
A few minor comments, but also noted a merge conflict.

File: python/transformations/Sentiment-Analysis/library.json (hunk 1 above)
Comment (SteveRosam) [reply python/transformations/Sentiment-Analysis/library.json:37]: sorry i should have done on a separate PR but yes let's leave here this time

File: python/transformations/Threshold-Alert/README.md (hunk 2 above)
Comment (SteveRosam) [reply python/transformations/Threshold-Alert/README.md]: done

File: python/transformations/Threshold-Alert/README.md (hunk 3 above)
Comment (SteveRosam) [reply python/transformations/Threshold-Alert/README.md]: done

**Review by tbedford — APPROVED** [review]

**Pull request merged**